www/_posts/2020-06-29-cordova-android-9.0.0.md

```diff
@@ -0,0 +1,159 @@
+
+By default, Cordova has Kotlin disabled but can be enabled with the `GradlePluginKotlinEnabled` `preference` tag.
+
+Additionally, Kotlin’s coding style and version are configurable. By default, Cordova sets the coding style as `official` and uses version `1.3.50`.
+
+Below is an example `config.xml` for enabling and configuring Kotlin.
+
+```xml
+<preference name="GradlePluginKotlinEnabled" value="true" />
+<preference name="GradlePluginKotlinCodeStyle" value="official" />
+<preference name="GradlePluginKotlinVersion" value="1.3.50" />
+```
+
+For plugin developers, it is recommended to ensure that the Kotlin files are placed into the platforms `src/main/kotlin` directory.
+
+* **Added AndroidX Support**
+
+AndroidX is the newest and improved namespace for the Android Support Libraries. The original support libraries are no longer maintained.
+
+It is recommended to use AndroidX so that your app is running the latest support libraries but, by default, Cordova has AndroidX support disabled.
+A lot of the Android supported plugins are still using the old support libraries which can not build when using the AndroidX support libraries. It is recommended to research each plugin to see if they support AndroidX before enabling this feature.
+
+It is recommended for plugin developers to start migrating to support AndroidX. App developers could also use Jetifier to automatically migrates their existing third-party libraries to use AndroidX.
+
+You can enable this feature by setting the preference flag `AndroidXEnabled` to `true`.
+
+```xml
+<preference name="AndroidXEnabled" value="true" />
+```
+
+If you were previously using the [cordova-plugin-androidx](https://www.npmjs.com/package/cordova-plugin-androidx) plugin to enable AndroidX support, this plugin is no longer needed with this preference flag.
+
+The [cordova-plugin-androidx-adapter](https://www.npmjs.com/package/cordova-plugin-androidx-adapter) plugin is still required to migrate the legacy references to the new AndroidX references.
+
+* **Added Google Services Support**
+
+To use Google APIs or Firebase services, the Gradle plugin [Google Services](https://developers.google.com/android/guides/google-services-plugin) is required to be enabled when building your Android application.
+
+For plugin developers, this can be enabled by setting to the app's `config.xml` the `GradlePluginGoogleServicesEnabled` `preference` flag. From the `plugin.xml` file, you can do this by adding the following lines:
+
+```xml
+<config-file target="config.xml" parent="/*">
+    <preference name="GradlePluginGoogleServicesEnabled" value="true" />
+    <preference name="GradlePluginGoogleServicesVersion" value="4.2.0" />
+</config-file>
+```
+
+* **Android Version Support Update**
+
+* The default target SDK version is set to 29.
+* The minimum SDK version is set to 22.
+
+**NOTE:** because Cordova has increased the minimum SDK version to 22, we **no longer support or test** with Android 5.0 or lower.
```

Comment:
Add "The minimum supported Android version is 5.1." because I'm sure people will misunderstand and expect it to support 5.0.